crates/proto/src/dnssec/dnssec_dns_handle/mod.rs

```diff
@@ -880,7 +860,6 @@ where
 
     // otherwise we need to recursively discover the status of DS up the chain,
     //   if we find a valid DS, then we're in a Bogus state,
-    //   if we find no records, then we are Indeterminate
```

Comment:
Does this mean we never construct `Indeterminate` anymore?

Reply:
Yes, the first two commits remove all places where we construct it. I think it would only make sense to use `Indeterminate` if we supported separate trust anchors for different zones, then we'd use it if the root zone has no trust anchor, until the point where we get to a zone that anchors an island of security.